runner/runner.go

```diff
@@ -847,19 +863,62 @@ func (r *Runner) RunEnumeration() {
 					}
 				}
 			}
-			row := resp.str
+
+			if notJsonOrCsv || r.options.OutputAll {
+				//remove default hashes for plain output.
```

Comment:
Let's try to fix this properly. I'd recommend removing this and addressing printing to stdout directly where the output string is being constructed, for example, in `runner/runner.go.analyze(...)` at line 1625 with something like the following code:

```go
...
hashesMap := make(map[string]interface{})
if scanopts.Hashes != "" {
	hs := strings.Split(scanopts.Hashes, ",")
	shouldOutputHashes := !(r.options.JSONOutput || r.options.OutputAll)
	if shouldOutputHashes {
		builder.WriteString(" [")
	}
	for index, hashType := range hs {
		...
		if hashBody != "" {
			hashesMap[fmt.Sprintf("body_%s", hashType)] = hashBody
			hashesMap[fmt.Sprintf("header_%s", hashType)] = hashHeader
			if shouldOutputHashes {
				if !scanopts.OutputWithNoColor {
					builder.WriteString(aurora.Magenta(hashBody).String())
				} else {
					builder.WriteString(hashBody)
				}
				if index != len(hs)-1 {
					builder.WriteString(",")
				}
			}
		}
	}
	if shouldOutputHashes {
		builder.WriteRune(']')
	}
}
...
```